packager/hls/base/simple_hls_notifier.cc

```diff
@@ -41,6 +42,22 @@ bool IsCommonSystemId(const std::vector<uint8_t>& system_id) {
          std::equal(system_id.begin(), system_id.end(), media::kCommonSystemId);
 }
 
+std::string Base64EncodeUri(const std::string& prefix,
```

Comment:
There is base::Base64UrlEncode() which does base64-url-encoding
https://en.wikipedia.org/wiki/Base64#URL_applications

So I prefer renaming this to Base64EncodeData().